### DISCUSSION TITLE
功能被插件冲突检测关掉怎么一键开启

### DISCUSSION THREAD

**Discussion by Paper-white** [FloatTech/ZeroBot-Plugin#305] [Q&A] [answered]
RT
群里被不知道谁拉了个同样功能的bot（对方甚至是默认配置啥都没改，吵得很）
我就试了一下插件冲突检测 结果把俩人的功能都给全关了
我要把我的全部打开怎么搞，总不能一个一个来启用把www

**Comment by fumiama** [comment, marked as answer]
目前只能如此。不介意丢失控制数据的话，可以删除data/control/下的数据库
